### ISSUE TITLE
<Dialog /> overlay doesn't disable the scroll at the top level when open.

### ISSUE THREAD

**Issue by bansalrachita** [microsoft/fluentui#12197]
### Environment Information
- Library: Fluent-ui-react
- **Package version(s)**:  ^0.44.0

Image Attached:
![image](https://user-images.githubusercontent.com/10090750/75936950-6db55000-5e38-11ea-9175-68f9519378dc.png)

Link to repro with an example:
https://codesandbox.io/s/fluent-ui-example-qq1qh

#### Actual behavior:

`< Dialog />`  doesn't disable the scroll at the root when open. Causes an awkward scrolling of the element in backdrop when ⬇️ ⬆️ keys are pressed.

#### Expected behavior:

Scroll should be disabled at the top element when dialog is open.

### Priorities and help requested:

Are you willing to submit a PR to fix? (Yes)

Requested priority: Normal

**Comment by aneeshack4** [comment]
Hi @bansalrachita, thanks for reaching out to us. I'm having trouble reproing this... as you can see in the screenshot below, when I press the up down arrows, the focus moves cleanly onto the buttons inside the Dialog: 
![image](https://user-images.githubusercontent.com/4161791/75963977-459b1080-5e7b-11ea-969c-6d49c1343a27.png)

**Comment by bansalrachita** [comment]
@aneeshack4, Sorry for explaining myself well, the screenshot I gave is taken through a screen reader extension [chromevox](https://chrome.google.com/webstore/detail/chromevox-classic-extensi/kgejglhpjiefppelpmljglcjbhoiplfn?hl=en) in Chrome. 

The issue arises when the overflow behavior of element in backdrop is scroll and the dialog is open. In the example, when you press the arrows, the element in the backdrop appears to start scrolling even though the focus is on the `cancel` button inside the `<Dialog />`. As you'd see the dialog height is very small, that fits well within the viewport and shouldn't get the scroll which it does when it's open.

**Comment by aneeshack4** [comment]
@bansalrachita Thanks for providing additional details. Still missing parts of the picture though... so this repros only when the chromevox screenreader is being used to navigate the page? Have you checked to see that it repros with Narrator? By "overflow behavior of element in backdrop is scroll", what do you mean... can you provide detailed repro steps, preferably with Narrator? Also, are you a dev inside Microsoft or out in OSS? I'm curious what context you're using this component in.

**Comment by bansalrachita** [comment]
Here's a video explaining the same: https://imgur.com/DFzh5rX

tl;dr: the backdrop scrolls when the dialog is open w/ or w/o a screen-reader. I've shown a video with screen-reader.

**Comment by aneeshack4** [comment]
I just ran Narrator on your codesandbox - this doesn't repro... even without screenreader, I'm not seeing a repro when the dialog is open and I scroll, the focus stays properly on the button inside the dialog - there's nothing in the backdrop that gets focus...

**Comment by bansalrachita** [comment]
I could see the same issue with the Narrator as well. The focus moves from the Dialog to the outside element. Pinged you in Teams as well.

**Comment by aneeshack4** [comment]
We spoke offline about this issue, this is going to be addressed shortly by the team who originally wrote these controls.

**Comment by jurokapsiar** [comment]
Tracking Reference: 6654
